UsageExamples/OoplesIndicatorLibraryExample.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Alpaca.Markets;
+using OoplesFinance.StockIndicators.Models;
+using static OoplesFinance.StockIndicators.Calculations;
+
+namespace UsageExamples
+{
+    public class OoplesIndicatorLibraryExample
+    {
+        const string paperApiKey = "REPLACEME";
+        const string paperApiSecret = "REPLACEME";
+        const string symbol = "AAPL";
+
+        public async Task RunExample()
+        {
+            var startDate = new DateTime(2021, 01, 01);
+            var endDate = new DateTime(2021, 12, 31);
+
+            var client = Environments.Paper.GetAlpacaDataClient(new SecretKey(paperApiKey, paperApiSecret));
+            var bars = (await client.ListHistoricalBarsAsync(new HistoricalBarsRequest(symbol, startDate, endDate, BarTimeFrame.Day)).ConfigureAwait(false)).Items;
+            var stockData = new StockData(bars.Select(x => x.Open), bars.Select(x => x.High), bars.Select(x => x.Low), bars.Select(x => x.Close), bars.Select(x => x.Volume), bars.Select(x => x.TimeUtc));
+
+            // can do typical indicators such as using typical bollinger band settings
+            var results = stockData.CalculateBollingerBands();
+
+            // we can also do completely custom indicators with very little effort such as below
+            //var results2 = stockData.CalculateBollingerBands(MovingAvgType.Ehlers3PoleSuperSmootherFilter, 50, 3);
+
+            // can also do indicators out of any indicators such as below
+            // we are getting the macd of a typical rsi
+            //var results3 = stockData.CalculateRelativeStrengthIndex().CalculateMovingAverageConvergenceDivergence();
+
+            var upperBandList = results.OutputValues["UpperBand"];
```

Comment:
Introduce constants for output series and use these constants later for output.

Reply:
I'm not familiar at all with the idea of a constant list and I tried to find examples and didn't find anything. Do you have pseudo code showing what you mean?

Reply:
Just constants for strings like `UpperBand`, `MiddleBand', and `LowerBand`. Ideally, these strings should be part of your library or your library should provide some strongly-typed wrapper for different output sequences but I'm OK with constants defined in this class for now.

Reply:
I understand what you mean now and as far as the output sequences in my library, I have been trying to think of methods on how to handle this but haven't come up with anything yet. I would love any ideas you might have since you have much more experience than I do